src/bin/vip-dev-env-import-media.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+/**
+ * @flow
+ * @format
+ */
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import command from '../lib/cli/command';
+
+command( {
+	requiredArgs: 1,
+} )
+	.option( 'slug', 'Custom name of the dev environment' )
+	.argv( process.argv, async ( unmatchedArgs: string[], opt ) => {
+		console.log( 'Importing media to a dev environment is not available yet.' );
```

Comment:
Maybe we should drop this if not supported? But I guess keeping it is fine as well.

Reply:
I'm working on adding this feature right now, so I think it's ok. We won't be releasing a new version of the CLI in the meantime.